eng/scripts/get_package_properties.py

```diff
@@ -1,11 +1,76 @@
 import argparse
-import sys
-import glob
 import os
 import re
 
+from typing import Dict, List
+
 from ci_tools.parsing import ParsedSetup
 
+additional_pr_triggers: Dict[str, List[str]] = {
+    "azure-core":[
+        os.path.join("/sdk", "servicebus", "azure-servicebus"),
+        os.path.join("/sdk", "eventhub", "azure-eventhub"),
+        os.path.join("/sdk", "tables", "azure-data-table"),
+        os.path.join("/sdk", "appconfig", "azure-appconfig"),
+        os.path.join("/sdk", "keyvault", "azure-keyvault-keys"),
+        os.path.join("/sdk", "identity", "azure-identity"),
+        os.path.join("/sdk", "core", "azure-mgmt-core"),
+        os.path.join("/sdk", "core", "azure-core-experimental"),
+        os.path.join("/sdk", "core", "azure-core-tracing-opentelemetry"),
+        os.path.join("/sdk", "core", "azure-core-tracing-opencensus"),
+        # todo: this currently won't be included, as azure-cosmos needs some special construction
+        # related to windows only + emulator
+        # os.path.join("/sdk", "cosmos", "azure-cosmos"),
+        os.path.join("/sdk", "ml", "azure-ai-ml"),
+        os.path.join("/sdk", "ai", "azure-ai-documentintelligence"),
+        os.path.join("/sdk", "ai", "azure-ai-inference"),
+        os.path.join("/sdk", "ai", "azure-ai-textanalytics"),
+        os.path.join("/sdk", "ai", "azure-ai-doctranslation"),
+        os.path.join("/sdk", "compute", "azure-mgmt-compute"),
+        os.path.join("/sdk", "communication", "azure-communication-chat"),
+        os.path.join("/sdk", "communication", "azure-communication-identity"),
+        os.path.join("/sdk", "storage", "azure-storage-blob")
+    ],
+    "azure-mgmt-core": [
+        os.path.join("/sdk", "compute", "azure-mgmt-compute"),
+        os.path.join("/sdk", "network", "azure-mgmt-network"),
+        os.path.join("/sdk", "resource", "azure-mgmt-resource"),
+        os.path.join("/sdk", "keyvault", "azure-mgmt-keyvault")
+    ]
+}
+
+# todo triggers based on paths and not files
```

Comment:
Let's try to get these triggers in place before merging this PR and disabling the other CI pipelines.

Reply:
Yeah this is the function I was mentioning before. Agreed.

Reply:
1000%. Doing some tests over the course of the evening.